### ISSUE TITLE
No support for aria-current in Edge

### ISSUE THREAD

**Issue by LJWatson** [FreedomScientific/VFO-standards-support#48]
### Summary
Jaws does not recognise the `aria-current` attribute in Edge.

### Expected result
Jaws should announce "current" when (actual or virtual) focus moves to an element with the `aria-current` attribute. where one of the tokens has been used (page, date, time, location, step), Jaws should append the token to the announcement.

### Actual result
Jaws does not announce current (with or without a token value).

### Example
http://design-patterns.tink.uk/aria-current/index.html

#### JAWS version and build number
Jaws 2018.1801.81

#### Operating System and version
Windows 10 64bit

#### Browser and version:
Edge 41

**Comment by scottaohara** [comment]
Note this is still an issue with JAWS 2018 August 2018 release.

**Comment by stevefaulkner** [comment]
Given that Edge in its current state [will not continue to be developed](https://twitter.com/stevefaulkner/status/1070761434450141185) closing this as won't fix
